snapcraft.yaml

```
@@ -422,9 +426,25 @@ parts:
           build_edk2 \
             "${CRAFT_PART_INSTALL}/share/qemu/OVMF_CODE.4MB.CSM.fd" \
             "${CRAFT_PART_INSTALL}/share/qemu/OVMF_VARS.4MB.CSM.fd" \
+            RELEASE \
             -DFD_SIZE_4MB \
             -DCSM_ENABLE=TRUE
       fi
+
+      # Debug firmware (4MB, debug mode)
+      # Set DEBUG_VERBOSE flag
+      if [ "$(uname -m)" = "aarch64" ]; then
+          sed -i "s#DEBUG_PRINT_ERROR_LEVEL = 0x8000004F#DEBUG_PRINT_ERROR_LEVEL = 0x8040004F#g" ArmVirtPkg/ArmVirt.dsc.inc
```

```suggestion
          sed -i "s#DEBUG_PRINT_ERROR_LEVEL = 0x8000004F#DEBUG_PRINT_ERROR_LEVEL = 0x8040004F#g" "${PKG}.inc"
```

Since we already have a `PKG` variable, it feels safer to make use of it to avoid anyone changing the variable and forgetting about the hardcoded reference.

---

It doesn't work, because names are different `ArmVirtPkg/ArmVirtQemu.dsc` and `ArmVirtPkg/ArmVirt.dsc`. There is an additional `Qemu` word.